woodwork/tests/accessor/test_statistics.py

```diff
@@ -228,7 +229,8 @@ def test_describe_accessor_method(describe_df):
     datetime_ltypes = [Datetime]
     formatted_datetime_ltypes = [Datetime(datetime_format='%Y~%m~%d')]
     timedelta_ltypes = [Timedelta]
-    numeric_ltypes = [Double, Integer]
+    nullable_numeric_ltypes = [Double, IntegerNullable]
```

Comment:
Do we need to test Mutual Info with `IntegerNullable` and `Integer`?

Reply:
Same question for Boolean and BooleanNullable

Reply:
I added a couple more tests cases for `make_categorical_for_mutual_info` and `replace_nans_for_mutual_info` to make sure they work with the different pandas dtypes. Our test fixture `df_same_mi` uses the `IntegerNullable` logical type, so I _think_ we are covered here.

@tamargrey Any thoughts since you are probably more familiar with the mutual info code than I am?

Reply:
I think the mutual info tests look good. (I guess we will only ever have to replace nans for BooleanNullable and IntegerNullable, so that's interesting), but I am wondering if we should have `boolean_ltypes = [Boolean, BooleanNullable]` or maybe `BooleanNullable` in its own list for the describe tests.

Reply:
Updated `describe_df` to include a missing value in the boolean data and added separate assertions for `Boolean` and `BooleanNullable` as with `Integer` and `IntegerNullable`.